src/main/resources/application.yml

```diff
@@ -37,10 +97,38 @@ resilience4j.circuitbreaker:
   instances:
     transactions-backend:
       baseConfig: default
+      ignoreExceptions:
+        - it.pagopa.transactions.exceptions.UnsatisfiablePspRequestException
+        - it.pagopa.transactions.exceptions.PaymentNoticeAllCCPMismatchException
+        - it.pagopa.transactions.exceptions.TransactionNotFoundException
+        - it.pagopa.transactions.exceptions.AlreadyProcessedException
+        - it.pagopa.transactions.exceptions.NotImplementedException
+        - it.pagopa.transactions.exceptions.InvalidRequestException
+        - it.pagopa.transactions.exceptions.TransactionAmountMismatchException
+        - it.pagopa.transactions.exceptions.NodoErrorException
+        - it.pagopa.transactions.exceptions.InvalidNodoResponseException
     node-backend:
       baseConfig: default
       ignoreExceptions:
+        - it.pagopa.transactions.exceptions.UnsatisfiablePspRequestException
+        - it.pagopa.transactions.exceptions.PaymentNoticeAllCCPMismatchException
+        - it.pagopa.transactions.exceptions.TransactionNotFoundException
+        - it.pagopa.transactions.exceptions.AlreadyProcessedException
+        - it.pagopa.transactions.exceptions.NotImplementedException
+        - it.pagopa.transactions.exceptions.InvalidRequestException
+        - it.pagopa.transactions.exceptions.TransactionAmountMismatchException
         - it.pagopa.transactions.exceptions.NodoErrorException
+        - it.pagopa.transactions.exceptions.InvalidNodoResponseException
     ecommerce-db:
       baseConfig: default
+      ignoreExceptions:
+        - it.pagopa.transactions.exceptions.UnsatisfiablePspRequestException
+        - it.pagopa.transactions.exceptions.PaymentNoticeAllCCPMismatchException
+        - it.pagopa.transactions.exceptions.TransactionNotFoundException
+        - it.pagopa.transactions.exceptions.AlreadyProcessedException
+        - it.pagopa.transactions.exceptions.NotImplementedException
+        - it.pagopa.transactions.exceptions.InvalidRequestException
+        - it.pagopa.transactions.exceptions.TransactionAmountMismatchException
+        - it.pagopa.transactions.exceptions.NodoErrorException
+        - it.pagopa.transactions.exceptions.InvalidNodoResponseException
```

Comment:
You could use an alias and an anchor if you want to avoid repeating the same list every time :)

Reply:
Unfortunately jackson dataformat [seems to not support anchors](https://github.com/FasterXML/jackson-dataformats-text/issues/98) so introducing anchors will break tests